### DISCUSSION TITLE
How to delete / cancel my Hubs Cloud AWS?

### DISCUSSION THREAD

**Discussion by tttigo** [mozilla/hubs#3097] [Hubs Cloud Questions] [answered]
Hey guys,

I've subscribed to Hubs Cloud AWS to do some successful tests and put it offline. But I want to know how exactly to cancel all AWS services to stop being charged.

I didn't find any doc dealing with that.

Thanks!!

**Comment by robinkwilson** [comment, marked as answer]
**To delete your hub stack + resources**
1. Go to AWS Console > **CloudFormation** > **Stacks** (make sure you're in the right region)
2. Select your Hubs Cloud stack
3. Press Delete
4. Confirm

**To delete your stored backups**
1. Go to AWS Console > **S3**
2. Look for hub backups (the folders would be named after your stack name) 
3. If there are, delete the folders + files.

After that you've successfully cancelled and deleted all of your Hub's resources!

**Reply by tttigo** [reply]
Thank you very much!! 

Can you tell me how long it takes to the billing page update?

**Reply by robinkwilson** [reply]
I'm not sure, that is an AWS question. I'd search: "how long does it take aws cost explorer to update?"

AWS cost explorer: https://console.aws.amazon.com/billing/home
